code/modules/mapfluff/ruins/spaceruin_code/listeningstation.dm

```diff
@@ -37,10 +37,17 @@
 	<br> It's not like anyone but me reads these, why else should I talk about my base upkeep? Today: the "lobby".
 "}
 
+/obj/item/paper/fluff/ruins/listeningstation/reports/june/goliath
```

Comment:
needs modular

Reply:
i surrounded it with comments since most of the code in this file is from /tg/, re-open review if insufficient